src/lipalightninglib.udl

```diff
@@ -253,7 +253,7 @@ interface LiquidityLimit {
 // Information embedded in an invoice
 dictionary InvoiceDetails {
     string invoice; // The BOLT11 invoice.
-    u64? amount_msat; // Payment amount in msats, if specified. If not available, invoice is an open-amount invoice
+    Amount? amount; // Payment amount, if specified. If not available, invoice is an open-amount invoice
```

Comment:
Now that `InvoiceDetails` carries a fiat value, we need to make clear which exchange rate is used for that fiat value. At the moment, I'm worried about the following situation in which a discrepancy can arise:
1. Consumer creates an invoice and looks at the fiat value contained within `InvoiceDetails`
    * It is reasonable for the consumer to expect that this is the fiat value that will get persisted in the local records
2. Later, the consumer gets the list of `Payments` and the `InvoiceDetails` inside the related payment has a different fiat value.

Maybe we don't need to worry about this edge case